Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:21.10
+
+#COPY CFG-Vault .
+#  Installs all dependances 
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get -y update && \
+    apt -y install \
+    e2fsprogs graphviz \
+    ruby-mustache \
+    squashfs-tools \
+    qemu-system-x86 \
+    qemu-utils \
+    python3  \
+    python3-pip \
+    wget \
+    net-tools
+    
+    
+# Adding tmux
+RUN pip3 install web.py 
+RUN pip3 install graphviz
```

Comment:
I don't think we depend on any graphviz support from Python, right?

Reply:
Took away the graphviz pip install